modules/core/core-api/src/main/java/com/enonic/xp/content/ProjectSyncParams.java

```diff
@@ -0,0 +1,65 @@
+package com.enonic.xp.content;
+
+import com.google.common.base.Preconditions;
+
+import com.enonic.xp.project.ProjectName;
+
+public final class ProjectSyncParams
+{
+    private final ProjectName sourceProject;
```

Comment:
I am not sure source project should be here.
1. It allows to sync content from any project
2. It does not fit into muli-inheritance 

Service should get sourceProject(s) from project data

`targetProject` should be called just `project`